### PR TITLE
Add libde265.

### PR DESCRIPTION
**Pull request by dlorenc**
I started poking at the dependency tree for imagemagick

### CODE REVIEW

File: libde265.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libde265/APKBUILD
```
Comment (tuananh) [inline libde265.yaml:1]: you wrote a tool to parse APKBUILD?? If you do, please do share it. I'm willing to be an alpha tester :D

Comment (dlorenc) [reply libde265.yaml:1]: @rawlingsj did! We're still testing it but it works pretty well :) He's going to get it cleaned up and released soon :)

Comment (tuananh) [reply libde265.yaml:1]: Haha i planned to build it too but never gotten around to it

Comment (tuananh) [reply libde265.yaml:1]: Amazing works btw @rawlingsj

**Review by imjasonh — DISMISSED** [review]

**Review by imjasonh — DISMISSED** [review]

File: libde265.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        - uses: split/dev
+      dependencies:
+        runtime:
+            - libde265
+      description: libde265 dev
+    - name: libde265-examples
+      pipeline:
+        - runs: |
+            mkdir -p "${{targets.subpkgdir}}"/usr/bin/
+            mv "${{targets.destdir}}"/usr/bin/ ${{targets.subpkgdir}}/usr/bin
```
Comment (jdolitsky) [inline libde265.yaml:62]: this looks a little strange.. wont this take /usr/bin/libde265-dec265 ?

Comment (imjasonh) [reply libde265.yaml:62]: +1, wdyt @dlorenc

**Pull request closed without merging**